ui/wdry.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const whyDidYouRender = require('@welldone-software/why-did-you-render');
+
+whyDidYouRender(React, { trackAllPureComponents: true });
```

Comment:
We need to add a `process.env` check here for performance but thus far I've not gotten it working.